### ISSUE TITLE
[Bug][Jupyter Widget] Python 3 compatibility?

### ISSUE THREAD

**Issue by darribas** [keplergl/kepler.gl#579]
**Describe the bug**
Jupyter widget for Kepler does not seem to work under Python 3

**To Reproduce**
1. Using the following `Dockerfile`

```Dockerfile
FROM darribas/gds:2.0

RUN pip install keplergl
```
The `darribas/gds:2.0` image is based on the Jupyter docker stacks, you can see versions installed [here](https://github.com/darribas/gds_env/blob/master/stack_py.txt).

2. Build the image

```
docker build -t kgl .
```

3. Launch JupyterLab through the container:

```
docker run -p 8888:8888 -v ${PWD}:/home/jovyan/work --rm -ti kgl
```

4. Open a new notebook and import Kepler.gl widget

<img width="957" alt="Capture" src="https://user-images.githubusercontent.com/417363/60164079-d6055980-97f4-11e9-8302-5fdf0bb20c2f.PNG">

Am I missing something?

**Comment by darribas** [comment]
Further inspecting, it seems to point to a bug on [L.56](https://github.com/keplergl/kepler.gl/blob/master/bindings/kepler.gl-jupyter/keplergl/keplergl.py#L56)?

**Comment by amir-ghasemi** [comment]
Replacing` print data` with `print(data)` resolved the issue.

**Comment by darribas** [comment]
In case it's of help, I've issued a patch with the fix over at #582 @heshan0131

**Comment by d-wasserman** [comment]
When should we expect this in the pip installation?

**Comment by heshan0131** [comment]
python3 fixes are released with 0.1.0a7. closing for now
